pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -619,6 +620,8 @@ func (m *kubeGenericRuntimeManager) getPodContainerStatuses(ctx context.Context,
 		return nil, nil, err
 	}
 
+	isManagedPod := utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling) && managed.IsManagedPodFromRuntimeService(ctx, m.runtimeService, activePodSandboxID)
```

Comment:
I don't think we need to check for feature gate here. `cStatus.Resources != nil` will only be true if the feature is enabled

Reply:
Addressed, thanks.